### ISSUE TITLE
The menu does not close if you click outside the menu.

### ISSUE THREAD

**Issue by iiigum** [FrDH/mmenu-light#13]
The menu does not close if you click outside the menu.
If you have created two menus on the page, then the last created one closes successfully, the first does not close.

**Comment by FrDH** [comment]
Hi,

Thanks for reporting this issue.
If you only create one menu, it does close?

**Comment by iiigum** [comment]
Yeah, that's all right.

**Comment by FrDH** [comment]
I'll have to do some testing (is there an online example of the issue?), but I think I know why this happens. The plugin creates a "blocker" DIV for each menu. These blocker DIVs are all active if any menu opens while only the corresponding blocker should be active.

**Comment by iiigum** [comment]
I used the example github files.
Here is an example (demo/default.html), nothing else has changed:
```
<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8" />
		<meta name="author" content="www.frebsite.nl" />
		<meta name="viewport" content="width=device-width initial-scale=1.0 maximum-scale=1.0 user-scalable=yes" />

		<title>mmenu light demo</title>

		<link type="text/css" rel="stylesheet" href="css/demo.css" />
		<link type="text/css" rel="stylesheet" href="../dist/mmenu-light.css" />
	</head>
	<body>
		<div id="page">
			<div class="header">
				<a menu href="#menu"><span></span></a>
				<a menu2 href="#menu2"><span></span></a>
				Demo
				<nav id="menu">
					<ul>
						<li class="Selected"><a href="#">Home</a></li>
						<li><span>About us</span>
							<ul>
								<li><a href="#about/history">History</a></li>
								<li><span>The team</span>
									<ul>
										<li><a href="#about/team/management">Management</a></li>
										<li><a href="#about/team/sales">Sales</a></li>
										<li><a href="#about/team/development">Development</a></li>
									</ul>
								</li>
								<li><a href="#about/address">Our address</a></li>
							</ul>
						</li>
						<li><a href="#contact">Contact</a></li>
					</ul>
				</nav>
				<nav id="menu2">
					<ul>
						<li class="Selected"><a href="#">Home</a></li>
						<li><span>About us</span>
							<ul>
								<li><a href="#about/history">History</a></li>
								<li><span>The team</span>
									<ul>
										<li><a href="#about/team/management">Management</a></li>
										<li><a href="#about/team/sales">Sales</a></li>
										<li><a href="#about/team/development">Development</a></li>
									</ul>
								</li>
								<li><a href="#about/address">Our address</a></li>
							</ul>
						</li>
						<li><a href="#contact">Contact</a></li>
					</ul>
				</nav>
			</div>

			<div class="content">
				<p><strong>This is a demo.</strong><br />
					Click the hamburger icon to open the menu.</p>
			</div>
		</div>

		<script src="../dist/mmenu-light.js"></script>
		<script>
			const menu = new MmenuLight( document.querySelector( '#menu' ), {
				 title: 'menu',
				 theme: 'light',
				 selected: 'Selected'
			});
			menu.enable( 'all' ); // '(max-width: 900px)'
			menu.offcanvas({
				position: 'left' ,
				move: true,
				blockPage: true 
			});

			//	Open the menu.
			document.querySelector( 'a[href="#menu"]' )
				.addEventListener( 'click', ( evnt ) => {
					menu.open();

					//	Don't forget to "preventDefault" and to "stopPropagation".
					evnt.preventDefault();
					evnt.stopPropagation();
				});

		</script>
		
				<script>
			const menu2 = new MmenuLight( document.querySelector( '#menu2' ), {
				title: 'menu2',
				theme: 'light',
				selected: 'Selected'
			});
			menu2.enable( 'all' ); // '(max-width: 900px)'
			menu2.offcanvas({
				position: 'right' ,
				move: true,
				blockPage: true
			});

			//	Open the menu.
			document.querySelector( 'a[href="#menu2"]' )
				.addEventListener( 'click', ( evnt ) => {
					menu2.open();

					//	Don't forget to "preventDefault" and to "stopPropagation".
					evnt.preventDefault();
					evnt.stopPropagation();
				});

		</script>
	</body>
</html>
```

**Comment by FrDH** [comment]
Fixed the issue in 2.0.3

**Comment by FrDH** [comment]
Should be fixed in version 2.1
